SignalServiceKit/src/Messages/Interactions/OWSContact.h

```diff
@@ -140,10 +141,19 @@ NSString *NSStringForContactAddressType(OWSContactAddressType value);
                                familyName:(nullable NSString *)familyName
                                nameSuffix:(nullable NSString *)nameSuffix;
 
+#pragma mark - Phone Numbers and Recipient IDs
+
+- (NSArray<NSString *> *)systemContactsWithSignalAccountPhoneNumbers:(id<ContactsManagerProtocol>)contactsManager
+    NS_SWIFT_NAME(systemContactsWithSignalAccountPhoneNumbers(_:));
+- (NSArray<NSString *> *)systemContactPhoneNumbers:(id<ContactsManagerProtocol>)contactsManager
+    NS_SWIFT_NAME(systemContactPhoneNumbers(_:));
+- (NSArray<NSString *> *)e164PhoneNumbers NS_SWIFT_NAME(e164PhoneNumbers());
+
 @end
 
 #pragma mark -
 
+// TODO: Move to separate source file, rename to OWSContactConversion.
 @interface OWSContacts : NSObject
```

Comment:
I don't think we should adopt the `Record` name convention within the codebase.  It's a ROR thing right?  These are model classes.  IMO we should either use Model or no suffix, and I vote no suffix.

Reply:
> OH also, there's a difference between OWSContact (the model) and OWSContacts

Oh! I didn't realized that OWSContacts was different from OWSContact. I agree with your suggested rename.

> I don't think we should adopt the Record name convention within the codebase. It's a ROR thing right? 

Since this was predicated on my misunderstanding, I'll save this discussion for another day. =)